src/styles/main.scss

```diff
@@ -321,3 +321,11 @@ div.rbt div {
   border-color: $nobel;
   background-color: $solitude;
 }
+
+.modal-open {
+  overflow: hidden;
+}
```

Comment:
this prevents the content in the main tag from scrolling when a modal is opened (and we add this class to the body tag as shown above)